saleor/plugins/webhook/plugin.py

```diff
@@ -11,7 +11,7 @@
 from ...core.taxes import TaxData, TaxType
 from ...core.utils.json_serializer import CustomJsonEncoder
 from ...payment import PaymentError, TransactionKind
-from ...payment.models import Payment
+from ...payment.models import Payment, TransactionItem
```

Comment:
Would be worth to add tests for logic added to this file.  (tests for `_trigger_metadata_updated_event`, and for each new handler added to WebhookPlugin).

Reply:
I've added tests for plugin manager methods, but I'm unsure about `_trigger_metadata_updated_event` - I would have to test it directly on webhook's plugin instance, or split into util function.